handlers/main.yml

```
@@ -2,9 +2,7 @@
 ---
 # Put handlers for tasks here.
 
-# Example:
-- name: template handler to restart services
+- name: restart postgresql
   service:
-    name: "{{ item }}"
+    name: "postgresql"
```

```suggestion
    name: postgresql
```
quotes not heeded here